api3/core/openapi.yaml

```diff
@@ -772,6 +783,42 @@ components:
         application/json:
           schema:
             $ref: "../common/openapi.yaml#/components/schemas/ApiResponse"
-  schemas: {}
+  schemas: 
+    APIClarityFeatureList:
+      type: 'object'
+      properties:
+        features:
+          type: array
+          items:
+            $ref: '#/components/schemas/APIClarityFeature'
+    APIClarityFeature:
+      type: 'object'
+      properties:
+        featureName:
+          $ref: '#/components/schemas/APIClarityFeatureEnum'
+        featureDescription:
+          description: "Short human readable description of the feature"
+          type: string
+        hostsToTrace:
```

Comment:
we should add a description to this. e.g. "List of hosts (with ports) that the feature requires to get trace for"

Reply:
its host:port
I think that we can put a description similar to the trace manager api:
https://github.com/openclarity/trace-sampling-manager/blob/main/api/swagger.yaml#L36

Reply:
I took a look at the envoy plugin code https://github.com/openclarity/wasm-filters/blob/0ae3f8fe2e4e89068c617d145b6086651ed48ac2/src/trace/main.go and I've found this. If I read it correctly this assumes that the hostsToTrace contain just the host part and not host:port. Is it correct?

`func (ctx *TraceFilterContext) shouldTrace() bool {
	if !ctx.isHostFixed {
		return true
	}
	if !ctx.enableTraceSampling {
		return true
	}

	_, foundInApiToTrace := ctx.hostsToTrace[ctx.Telemetry.Request.Host]
	// check if we should trace all hosts
	_, shouldTraceAll := ctx.hostsToTrace["*"]

	if !foundInApiToTrace && !shouldTraceAll {
		proxywasm.LogDebugf("Host should not be traced. host=%v, foundInApiToTrace=%v, shouldTraceAll=%v",
			ctx.Telemetry.Request.Host, foundInApiToTrace, shouldTraceAll)
		return false
	}
	proxywasm.LogDebugf("Host should be traced. host=%v, foundInApiToTrace=%v, shouldTraceAll=%v",
		ctx.Telemetry.Request.Host, foundInApiToTrace, shouldTraceAll)
	return true
}`

Reply:
OK, making the needed changes to handle host:port